src/snapred/ui/workflow/ReductionWorkflow.py

```diff
@@ -280,46 +528,67 @@ def onArtificialNormalizationValueChange(self, smoothingValue, lss, decreasePara
         )
 
         response = self.request(path="reduction/artificialNormalization", payload=request_)
+
+        #
+        # TODO: why isn't this checking that the request actually succeeded?
```

Comment:
I think the assumption is that if there was an error, the code would go through the error handling routine instead of just returning.  
There is some code duplication here, I dont see any reason one couldnt call the other to some degree.